### PR TITLE
fix(slugification): add number support

### PR DESCRIPTION
**Pull request by kibblerz**
Adding support for numbers in the slug. It's a simple change, but quite useful to be able to use numbers in the slug fields.

### CODE REVIEW

**Comment by ComfortablyCoding** [comment]
Can you explain further what this should be accomplishing? Using a number field as a reference is already supported.

**Comment by kibblerz** [comment]
It does not, the slug ends up empty, or the reference field is skipped if skipUndefinedReferences is enabled. Numbers do not have a length property, so it is seen as undefined.

**Comment by ComfortablyCoding** [comment]
Alright, makes sense. Thanks for catching and reporting it. Will review it over the weekend.

File: server/services/slug-service.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = ({ strapi }) => ({
 
 		// ensure the reference field has data
 		let referenceFieldValues = references
-			.filter((r) => typeof data[r] !== 'undefined' && data[r].length)
+			.filter((r) => (typeof data[r] !== 'undefined' && data[r].length) || (typeof data[r] === 'number'))
```
Comment (ComfortablyCoding) [inline server/services/slug-service.js:23]:
```suggestion
			.filter((r) => (typeof data[r] !== 'undefined' && (data[r].length || typeof data[r] === 'number'))
```

**Review by ComfortablyCoding — CHANGES_REQUESTED** [review]

**Comment by ComfortablyCoding** [comment]
@kibblerz ,

Please resolve the suggested change and remove the changes to lines 36 & 38. Once completed I will merge.

**Comment by ComfortablyCoding** [comment]
Closing as this will be fixed by #78

**Pull request closed without merging**